assets/javascripts/discourse/components/chat-message.js

```diff
@@ -91,15 +91,15 @@ export default Component.extend({
     return deletedAt && !expanded;
   },
 
-  @computed(
+  @discourseComputed(
     "selectingMessages",
     "canFlagMessage",
     "showDeleteButton",
     "showRestoreButton",
     "showEditButton",
     "showRebakeButton"
   )
-  get moreButtons() {
+  moreButtons() {
```

Comment:
Did you get any input on what we should do? Favouring `@discourseComputed` vs `@computed get`?

Reply:
Yeah CVX said to use `@discourseComputed`...maybe I will add a post in our lore topic about it.